### ISSUE TITLE
[GS v4] hhea table diff

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/googlesans#222]
Review diff of hhea tables between David Berlow's builds @ googlefonts/GoogleSans-fb@a8d2737 and our v3.003 production releases

**Comment by chrissimpkins** [comment]
Roman var font

<details>
<summary> diff</summary>

```diff
--- A/GoogleSans[opsz,wght].ttf	2020-09-16T16:50:43-04:00
+++ B/GoogleSansGrades-VF.ttf	2021-02-25T11:12:08.527802-05:00
@@ -8,7 +8,7 @@
     <lineGap value="0"/>
     <advanceWidthMax value="3284"/>
     <minLeftSideBearing value="-354"/>
-    <minRightSideBearing value="-527"/>
+    <minRightSideBearing value="-256"/>
     <xMaxExtent value="3284"/>
     <caretSlopeRise value="1"/>
     <caretSlopeRun value="0"/>
@@ -18,7 +18,7 @@
     <reserved2 value="0"/>
     <reserved3 value="0"/>
     <metricDataFormat value="0"/>
-    <numberOfHMetrics value="2545"/>
+    <numberOfHMetrics value="2642"/>
   </hhea>
```
</details>

**Comment by chrissimpkins** [comment]
Italic var font

<details>
<summary>diff </summary>

```diff
--- A/GoogleSans-Italic[opsz,wght].ttf	2020-09-16T16:50:43-04:00
+++ B/GoogleSansItalicGrades-VF.ttf	2021-02-25T11:12:48.431719-05:00
@@ -8,7 +8,7 @@
     <lineGap value="0"/>
     <advanceWidthMax value="3284"/>
     <minLeftSideBearing value="-314"/>
-    <minRightSideBearing value="-492"/>
+    <minRightSideBearing value="-489"/>
     <xMaxExtent value="3284"/>
     <caretSlopeRise value="1000"/>
     <caretSlopeRun value="176"/>
@@ -18,7 +18,7 @@
     <reserved2 value="0"/>
     <reserved3 value="0"/>
     <metricDataFormat value="0"/>
-    <numberOfHMetrics value="2539"/>
+    <numberOfHMetrics value="2638"/>
   </hhea>

```

</details>

**Comment by chrissimpkins** [comment]
Reviewed again as of https://github.com/googlefonts/googlesans/pull/240/commits/b26739deb19ca6740903a69df7499c8851ff868a.  No concerns.  This issue will be closed when #240 is merged
